### ISSUE TITLE
How to check validation loss????

### ISSUE THREAD

**Issue by shinpaul14** [open-mmlab/mmsegmentation#3438]
Hello,
I am just wondering how to get validation loss in the current version where previous issues mentioned val_step or workflow.
But all of those do not work now, so I was wondering how can we get validation loss or how can we check the dice score in our training step???

**Comment by shinpaul14** [comment]
I am using mmseg version 1.2.1 
#1396 I did loo in to this code but in the version I am using does not have Val_step to add loss in to the log.

**Comment by ly27253** [comment]
我也注意到这个问题了，我迫切希望知道将怎么获取验证损失，希望通过训练损失和验证损失查看模型是否欠拟合了
